### ISSUE TITLE
MS Edge on Linux does not trust certificate.

### ISSUE THREAD

**Issue by themooserooster** [tmds/linux-dev-certs#29]
Awesome tool, but I had to run 

```bash
certutil -d sql:$HOME/.pki/nssdb -A -t "C,," -n aspnetcore-edge \ -i /etc/local/share/ca-certificates/aspnet-dev-<MY_USERNAME>.crt
```

as an extra step to get Edge on Linux to trust the certificate. Not sure if this affects other Chromium browsers as well. Running Ubuntu 24.04 LTS.

**Comment by tmds** [comment]
Yes, as mentioned on the README, only Ubuntu only Firefox is currently configured.

Do you want to add support for MS Edge? If not, I'll look into it when I find some time.

**Comment by asvishnyakov** [comment]
@themooserooster @tmds As for Chrome, I just opened chrome://settings/certificates and manually imported CA certificate from /usr/local/share/ca-certificates. Works fine.

**Comment by Xupack88** [comment]
> @themooserooster @tmds As for Chrome, I just opened chrome://settings/certificates and manually imported CA certificate from /usr/local/share/ca-certificates. It even automatically pop up in the dialog as "recent file". Works fine.

On Kubuntu 24.04 using Thorium Browser after running "dotnet linux-dev-certs install" I had to go to "chrome://settings/certificates" and in there to "Authorities" tab as I found out from this SO post: [https://superuser.com/questions/1213287/private-key-is-missing-or-invalid-when-importing-a-certificate-in-google-chrom/1276793#1276793](url) and then import "/etc/ssl/certs/aspnet-dev-username.pem" and check "Trust this certificate for identifying websites" and after that no more errors.

![Screenshot_20240817_094234](https://github.com/user-attachments/assets/b09529b7-8493-47f2-88fa-9449cc9afe4a)
